README.md

```diff
@@ -1,125 +1,118 @@
 ```
 
-To uninstall nvm, just run:
+Install an older LTS release by codename.
 
-```
-rm -f $XDG_CONFIG_HOME/fish/{conf.d,functions,completions}/nvm.fish && emit nvm_uninstall
+```console
+nvm install carbon
 ```
 
-</details>
+> Installs `8.16.2`, the latest release of the Carbon LTS line.
 
-## Quickstart
-
-Download and switch to the latest Node.js release.
+Or install a specific version of Node.
 
 ```console
-nvm use latest
+nvm install v12.9.1
 ```
 
-> **Note:** This downloads the latest Node.js release tarball from the [official mirror](https://nodejs.org/dist), extracts it to <code>[\$XDG_CONFIG_HOME](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables)/nvm</code> and modifies your `$PATH` so it can be used immediately. Learn more about the Node.js release schedule [here](https://github.com/nodejs/Release).
+> Supports full or partial version numbers, starting with an optional "v".
 
-Download and switch to the latest LTS (long-term support) Node.js release.
+The `nvm install` command activates the specified Node version only in the current environment. If you want to set the default version for new shells use:
 
-```console
-nvm use lts
+```fish
+set -U nvm_default_version v12.9.1
 ```
 
-You can create a `.nvmrc` file in the root of your project (or any parent directory) and run `nvm` to use the version in it. `nvm` will try to find the nearest `.nvmrc` file, traversing the directory tree from the current working directory upwards.
+Activate a version you've already installed.
 
 ```console
-node -v > .nvmrc
-nvm
+nvm use lts
 ```
 
-Run `nvm` in any subdirectory of a directory with an `.nvmrc` file to switch to the version from that file. Similarly, running `nvm use <version>` updates that `.nvmrc` file with the specified version.
+List which versions you have installed (including the system Node if there is one).
 
 ```console
-├── README.md
-├── dist
-├── node_modules
-├── package.json
-└── src
-    └── index.js
+$ nvm list
+     system  
+    v8.17.0 lts/carbon 
+    v12.9.1  
+ ▶ v14.15.1 lts/fermium 
+    v15.3.0 latest
 ```
 
+Or list all the Node versions available to install.
+
 ```console
-echo lts >.nvmrc
-cd src
-nvm
-node -v
-v10.15.1
+$ nvm list-remote
+   ...
+   v14.13.1  
+   v14.14.0  
+   v14.15.0 lts/fermium 
+ ▶ v14.15.1 lts/fermium ✓
+    v15.0.0  
+    v15.0.1  
+    v15.1.0  
+    v15.2.0  
+    v15.2.1  
+    v15.3.0 latest ✓
 ```
 
-### Listing versions
-
-List all the supported Node.js versions you can download and switch to.
+Want to remove a Node version? You can do that too.
 
 ```console
-nvm ls
+nvm remove v12.9.1
 ```
 
-```console
-...
-10.14.2    (lts/dubnium)
-10.15.0    (lts/dubnium)
- 11.0.0
- 11.1.0
- 11.2.0
- 11.3.0
- 11.4.0
- 11.5.0
- 11.6.0
- 11.7.0    (latest/current)
+If you would like to use a different mirror of the Node binaries, for example, if you're behind a firewall, use:
+
+```fish
+set -g nvm_mirror https://npm.taobao.org/mirrors/node
```

Comment:
Should a command exist for this (similar to my comment about the universal version). The reason is that these things are not that discoverable otherwise. Whilst doing `nvm <tab>` will show immediately what is available, with descriptions.

Reply:
We could document them in `nvm -h`, though.

Reply:
That sounds like a good compromise.